src/accountant.ts

```diff
@@ -79,50 +161,36 @@ export class Accountant {
         return this.client.send(tx.sender.keystore, tx.receiver.address, amount);
     }
 
-    private async processClaim(claim: Claim): Promise<void> {
+    /* eslint-disable  @typescript-eslint/no-explicit-any */
+    private async handleConnectionRetries(f: { (): any },alias: string): Promise<void> {
+      let attempts = 0
+      const maxAttempts = 5
+      for(;;){
```

Comment:
is there no regular while loop? having the looping variable defined before the loop makes this a bit unreadable. Can you do a regular:
`for (let attempts = 0; attempts < maxAttempts; attempts++)`
Also maybe make the maxAttempts a configurable constant with a default of 5

Reply:
it was originally a `while(true)` loop but the linter coverted it to a `for(;;)`
The semantic is:
- never ending loop
- until you finish or you reach the max number of retries due to errors

This cannot be done `for (let attempts = 0; attempts < maxAttempts; attempts++)` (different semantic, retries should be handled by the catch), instead you could do `for (let attempts = 0; attempts < maxAttempts; {})` (attempts has to be increased inside the catch section). I find this less readable and wrong though, since the `attempts < maxAttempts` is not a condition that the for loop should take care of

wdyt ?

I'll make the max retries configurable anyway 👍

Reply:
retry policy is now configurable